.github/workflows/Ubuntu-22.yaml

```diff
@@ -0,0 +1,27 @@
+# GitHub Action Workflow for building the Fedora 35 images.
```

Comment:
Can we make sure to add this new image to the root readme list along with the build status?

Reply:
I'll do that.